tests/generation/test_utils.py

```diff
@@ -1647,6 +1647,37 @@ def test_new_cache_format(self, num_beams, do_sample):
                         )
                     )
 
+    def test_generate_with_quant_cache(self):
```

Comment:
this test would also require quanto, @SunMarc do you think it is possible to run these extra quanto tests with `pytest -k "test_generate_with_quant_cache" tests/models/` inside our quantization workflow for quanto? what do you think should be the best approach here?

Reply:
This one is a bit tricky. I think that launching these extra tests on only one job can be complicated for our runners and create unexpected errors, and we don't really want to spawn + 100 jobs just like transformers CI for now. So maybe add quanto as a dependency of the daily model CI (just like `bitsandbytes`) or remove these tests and add more specific tests inside the quanto_integration folder. WDYT @ydshieh ?

Reply:
Yes, so far the best (or simple) approach is just to add the dependency. (is it just one qunto lib or many of them to be installed?)

Reply:
just quanto !

Reply:
Great, l added a decorator for all tests that quanto is required. Let me know if that is not the best strategy :)

Reply:
Great thanks @zucchini-nlp ! I think so far the best would be to move: https://github.com/huggingface/transformers/pull/30483/files#r1587473765 into: https://github.com/huggingface/transformers/blob/main/tests/quantization/quanto_integration/test_quanto.py 🙏

Reply:
and add quanto installation here: https://github.com/huggingface/transformers/blob/main/docker/transformers-all-latest-gpu/Dockerfile